tools/linux-desktop-integrate.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+#
+# Run this script to create desktop entry for the development version
+# of XOD IDE. XOD file types will be associated with the development
+# version too.
+#
+
+set -e
+
+# The first is for per-user, the second is system-wide
```

Comment:
Maybe better accept an option "--system -s" to run it with desired paths easily?